### PR TITLE
Markov trend

### PR DESCRIPTION
**Pull request by javisantana**
this adds the temporal simluation analysis (aka spatial markov trend)

It's not totally finished, it lacks of column selector widget (cc @javierarce) but it's good enough to start testing. I found some problems when running it over some datasets and it'd be nice to test it so I'd merge asap, I'll update it when we other stuff is ready

CR @xavijam

### CODE REVIEW

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/spatial-markov-trend.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
+var BaseAnalysisFormModel = require('./base-analysis-form-model');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-form-models/spatial-markov-trend.js]: spatial-markov-trend-model.js 🙏  please

**Comment by xavijam** [comment]
Good to go! But please, if you don't mind, add `-model` to markov analysis model, merge with master, 🍏 , 🇪🇸 , 💯 .

File: lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-forms-collection.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ var PointInPolygonFormModel = require('./analysis-form-models/point-in-polygon-f
 var UnknownTypeFormModel = require('./analysis-form-models/unknown-type-form-model');
 var WeightedCentroidModelFormModel = require('./analysis-form-models/weighted-centroid-form-model');
 var SamplingFormModel = require('./analysis-form-models/sampling-form-model');
-var SpatialMarkovTrend = require('./analysis-form-models/spatial-markov-trend');
+var SpatialMarkovTrend = require('./analysis-form-models/spatial-markov-trend-model');
```
Comment (xavijam) [inline lib/assets/javascripts/cartodb3/editor/layers/layer-content-views/analyses/analysis-forms-collection.js]: ❤️  :kiss:

**Comment by javisantana** [comment]
hasta la misma polla de travis

**Pull request merged**